leetcode/023.swapPairs/swapPairs.py

```diff
@@ -0,0 +1,40 @@
+#leetcode: https://leetcode.com/problems/swap-nodes-in-pairs/description/
+
+def swapPairs(self, head):
+    """
+    :type head: ListNode
+    :rtype: ListNode
+    """
+    # time: O(n), space: O(1)
+    if head == None or head.next == None:
+        return head
+    res = pre = ListNode(0)
```

Comment:
rename res to `sentry` as https://codereview.stackexchange.com/questions/73304/the-benefits-of-sentry-in-a-linked-list